### ISSUE TITLE
There seems to be a problem with this challenge

### ISSUE THREAD

**Issue by AminuSufi585** [freeCodeCamp/freeCodeCamp#15259]
Challenge [use-a-retina-image-for-higher-resolution-displays](http://beta.freecodecamp.com/en/challenges/responsive-web-design-principles/use-a-retina-image-for-higher-resolution-displays) has an issue.
User Agent is: <code>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html


<style>
  img { height: 100px; width: 100px; }
</style>

<img src="https://s3.amazonaws.com/freecodecamp/FCCStickers-CamperBot200x200.jpg" alt="freeCodeCamp sticker that says 'Because CamperBot Cares'">

```

**Comment by darpots** [comment]
code doesnt seem to pass.

<style>
  img {
    height: 100px;
    width: 100px;
  }
</style>
![screen shot 2017-06-12 at 21 28 09](https://user-images.githubusercontent.com/5114671/27053689-104e7c54-4fb6-11e7-9e4a-0aff8540e278.png)

**Comment by systimotic** [comment]
Thanks for looking into this! We recently fixed this (#15775), so I'm closing this issue.

Happy coding!
